### ISSUE TITLE
ModuleNotFound error with latest release

### ISSUE THREAD

**Issue by epwalsh** [mosaicml/composer#2030]
**Environment**

mosaicml==0.13.0

Running `composer_collect_env` fails with:

```
❯ composer_collect_env
Traceback (most recent call last):
  File "/Users/evanw/.virtualenvs/llm/bin/composer_collect_env", line 5, in <module>
    from composer.utils.collect_env import main
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/__init__.py", line 10, in <module>
    from composer.trainer import Trainer
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/trainer/__init__.py", line 6, in <module>
    from composer.trainer.trainer import Trainer
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/trainer/trainer.py", line 33, in <module>
    from composer.callbacks import CheckpointSaver, OptimizerMonitor
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/callbacks/__init__.py", line 12, in <module>
    from composer.callbacks.health_checker import HealthChecker
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/callbacks/health_checker.py", line 20, in <module>
    from slack_sdk.webhook import WebhookClient
ModuleNotFoundError: No module named 'slack_sdk'
```

**To reproduce**

1. Install `mosaicml==0.13.0`
2. Run `python -c 'from composer.metrics.nlp import Perplexity'` (or something similar)

You'll get the same stacktrace as above:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/__init__.py", line 10, in <module>
    from composer.trainer import Trainer
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/trainer/__init__.py", line 6, in <module>
    from composer.trainer.trainer import Trainer
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/trainer/trainer.py", line 33, in <module>
    from composer.callbacks import CheckpointSaver, OptimizerMonitor
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/callbacks/__init__.py", line 12, in <module>
    from composer.callbacks.health_checker import HealthChecker
  File "/Users/evanw/.virtualenvs/llm/lib/python3.10/site-packages/composer/callbacks/health_checker.py", line 20, in <module>
    from slack_sdk.webhook import WebhookClient
ModuleNotFoundError: No module named 'slack_sdk'
```

**Comment by dakinggg** [comment]
Hey Pete, thanks for reporting this. We made a mistake in the release. You can work around it by using `pip install mosaicml[all]` and we will fix it very shortly :)

**Comment by epwalsh** [comment]
Thank you!

**Comment by dakinggg** [comment]
Sorry, shouldn't have been closed yet until we re-release :)

**Comment by dakinggg** [comment]
We've released `0.13.1` which should resolve this issue :)
